dnf-behave-tests/features/plugins-core/config-manager.feature

```diff
@@ -11,6 +11,11 @@ Background:
         | enabled     | 0        |
 
 
+Scenario: when run without arguments
+   When I execute dnf with args "config-manager"
+   Then the exit code is 1
+
```

Comment:
Can you please also check if stderr contains the message about required arguments?